packages/backend/src/payment-method/ilp/connector/core/middleware/account.ts

```diff
@@ -29,17 +29,25 @@ export function createAccountMiddleware(serverAddress: string): ILPMiddleware {
         )
       } catch (err) {
         // Don't complain if liquidity account already exists.
-        if (err instanceof AccountAlreadyExistsError) {
-          // Do nothing.
-        } else {
+        if (!(err instanceof AccountAlreadyExistsError)) {
```

Comment:
I can't comment on the exact line where this could be added, but in the spirit of the directive in https://github.com/interledger/rafiki/issues/2489  for "more is more", but might it be useful for a debug or trace-level log when liquidity accounts are successfully created